### PR TITLE
made some few tips/changes

### PR DESCRIPTION
**Pull request by ballyalley-o**
Great work, Kim!, love the retro feel! try to practice to target the classes and ids in for your css styling and event listeners. :)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,57 +1,60 @@
 // Player Name
-var diceNum = 1;
```
Comment (ballyalley-o) [inline app.js:2]: try to avoid using var and use the ES6's let/const, unless its for a purpose. :)

File: app.js [hunk 2]
```diff
@@ -1,57 +1,60 @@
...
 
 function rollTheDice() {
   count = count -1; //as when it will choose the timer number, it will be reduce as -1(because of reverse)
-    
+
   if(count != 0) {
       //Dice Picture 1
       diceNum = diceNum + 1;
       if(diceNum == 7) {
-             diceNum = 0;
```
Comment (ballyalley-o) [inline app.js:16]: your image ref: sai_1.gif is missing from the repo folder.

File: app.js [hunk 3]
```diff
@@ -1,57 +1,60 @@
...
   //change the dice picture2
-  var imgsrc2 = document.getElementsByTagName("img")[1];
-  var str2 = "./images/sai_"+num2+".gif";
+  let imgsrc2 = document.getElementsByTagName("img")[1];
+  let str2 = "./sai_"+num2+".gif";
   imgsrc2.src = str2;
   }
 }
-function randomDice() {
+
```
Comment (ballyalley-o) [inline app.js:50]: added an event listener instead of going inline func. :)

**Review by ballyalley-o — COMMENTED** [review]
OVERALL, you did a GREAT JOB, KIM! love the retro LOOK!

File: app.js (hunk 1 above)
Comment (hoiwonkim) [reply app.js:2]: I was confusing which one I need to use.
As "var" is old style but still we can use it. However, I learned the var during our class. So I just followed it.
"let" can update the values "const" can`t update. will not be changed.  Thank you for the advice. I feel comfortable using the let and Const in future work.

**Comment by hoiwonkim** [comment]
It is my first time doing a project with coding, so I was quite confused.
Thank you for the check and advice.
It was very helpful to find what I need to care about and need to study more.
Please keep helping me and providing good advice to improve my skill.
Thank you for your support.

**Comment by ballyalley-o** [comment]
> It is my first time doing a project with coding, so I was quite confused. Thank you for the check and advice. It was very helpful to find what I need to care about and need to study more. Please keep helping me and providing good advice to improve my skill. Thank you for your support.

Overall Kim, you did an amazing work!  :)

**Pull request merged**